frontend/.env.example

```diff
@@ -105,4 +105,8 @@ MOCKS_ENABLED=true
 
 # Allowed OIDC redirects -- list of allowed OIDC redirect URLs when mocking RAOIDC
 # (optional; default: http://localhost:3000/auth/callback/raoidc)
-MOCK_AUTH_ALLOWED_REDIRECTS=http://localhost:3000/auth/callback/raoidc
\ No newline at end of file
+MOCK_AUTH_ALLOWED_REDIRECTS=http://localhost:3000/auth/callback/raoidc
+
+# SCCH base URL -- used as a base URL for all SCCH calls
+# (required; use the example below)
+SCCH_BASE_URI=http://www.example.com/
```

Comment:
I wonder if we should start explicitly stating whether or not URLs in `env` need to have a trailing slash or not.

I think we should try to be consistent, at least. Maybe we could write some utility functions to either add or remove trailing slashes in the code so that the configuration is more flexible.